pkg/logql/log/fmt.go

```diff
@@ -43,6 +44,10 @@ var (
 			r := regexp.MustCompile(regex)
 			return r.ReplaceAllLiteralString(s, repl)
 		},
+		"count": func(substr string, s string) string {
+			count := strings.Count(s, substr)
```

Comment:
I wonder if we should support a regex pattern here as well?

Reply:
Yes we can. It's been added.